handlers/api.go

```diff
@@ -1205,7 +1205,29 @@ func getRocketpoolValidators(queryIndices []uint64) ([]interface{}, error) {
 }
```

Comment:
I just did some formating on these 2 queries because you cannot read it if it is just one line.
No functionality change.